src/eip4844/eip4844.c

```diff
@@ -106,9 +103,10 @@ static C_KZG_RET fr_batch_inv(fr_t *out, const fr_t *a, int len) {
 
     blst_fr_eucl_inverse(&accumulator, &accumulator);
 
-    for (i = len - 1; i >= 0; i--) {
-        blst_fr_mul(&out[i], &out[i], &accumulator);
-        blst_fr_mul(&accumulator, &accumulator, &a[i]);
+    for (size_t i = len; i > 0; i--) {
+        size_t index = i - 1;
+        blst_fr_mul(&out[index], &out[index], &accumulator);
+        blst_fr_mul(&accumulator, &accumulator, &a[index]);
```

Comment:
Hmm. IMO it's confusing to have one loop index, and then edit it first thing inside the loop.  Can we just leave the code as it was (but with size_t)? It's 4844 code as well, so I like not changing it if it's not broken.

Reply:
Thanks for the fix.